docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ros:humble-ros-base-jammy
+
+# change default shell to bash
+SHELL ["/bin/bash", "-c"]
+
+ARG FRI_CLIENT_VERSION=1.15
+
+RUN apt update -y 
+
+# create workspace
+RUN mkdir -p home/ros2_ws/src
+WORKDIR /home/ros2_ws/
+
+COPY ./src /home/ros2_ws/src
+RUN rosdep install -i --from-paths src --rosdistro ${ROS_DISTRO} -y 
+
+# "--symlink-install" allows the code in the locally mounted volume ./src to be adjusted without rebuilding
+RUN source /opt/ros/${ROS_DISTRO}/setup.bash && \
```

Comment:
okay so I cleaned some dependencies, as these are mostly in the base images already.

Couple of questions @Nicolai-98:
- I don't quite understand why sourcing is required here
- I don't use Docker a lot, so please let me know if you need scipy / numpy / vim / python3 is python etc here. So it is more convenient for your use

Reply:
Sorry I went ahead and merged this anyways already. Thank you again! You are now listed among contributors. 

Please feel free to create new PRs against `humble` if you see fit.